### DISCUSSION TITLE
Incompatibility with AVG Antivirus

### DISCUSSION THREAD

**Discussion by engerex** [rustdesk/rustdesk#7182] [General]
### Bug Description

When the AVG Antivirus window appears on the remote computer, keyboard and mouse control is not possible with RustDesk. AVG Antivirus is fine when the display is not front-facing. There is no problem when AVG Antivirus is running in the background.
  There are no problems with other remote connection applications.

### How to Reproduce

Install AVG Antivirus on remote desktop. and make a window appear on the screen (for example main window or scan window or other).

### Expected Behavior

working smoothly

### Operating system(s) on local side and remote side

Windows 11

### RustDesk Version(s) on local side and remote side

1.2.4

### Screenshots

no need 
![image](https://github.com/rustdesk/rustdesk/assets/5686053/244c6b73-2c22-44d1-b9c0-8a0500434e6c)

**Comment by fenisteel** [comment]
This issue is still exist, any AVG window blocks every input. The only way to get out of this lock is to send a Ctrl-Alt-Del to the client then open Task Manager, from which you can switch to another non-AVG window. If the taskbar thumbnails are enabled, you can hover your mouse over the AVG taskbar button and close the window from the thumbnail view, however this isn't work if you have a notification of a detected virus and you want to perform an action within the AVG window (ignore, move to quarantine, etc.). We use the Individual self-hosting Pro plan.

**Comment by trekfan1** [comment]
Same problem with Norton and, I think, with all security software
